core/files/css/base.php

```diff
@@ -327,6 +327,8 @@ public function add_control_rules( array $control, array $controls_stack, callab
 
 		$stylesheet = $this->get_stylesheet();
 
+		$control = apply_filters( 'elementor/files/css/selectors', $control, $value ?? [] );
```

Comment:
Remove

Reply:
There is still a need to use this filter. 
In this filter we add the css var "--e-bg-lazyload" to the selector list to be rendered inside the post css file.

Reply:
@matipojo 

This hook is no good for us, It's read only Class object, no setters. 
I found this file where I can add filter, This file : 

https://github.com/elementor/elementor/blob/e0f0db16123c2e9fa5854bb825844c399adf90c4/core/files/css/post.php#L298

The code is simple this is just a demo, with no filter.

`
$background_image = Utils::get_array_value_by_keys( $element_settings, [ 'background_image', 'url' ] );

if ( $background_image ) {
         $element_settings['--e-bg-lazyload'] = $background_image;
}`

But the issue with this code is that it won't parse this field as a CSS property because it's not defined as `style_fields` in : 

https://github.com/elementor/elementor/blob/e0f0db16123c2e9fa5854bb825844c399adf90c4/includes/base/controls-stack.php#L732

And to make it `style_fields` it means to update more files with this setting. Also, it will create the full selector again, in the CSS file. (maybe I'm wrong about this one. ) What do you think?

Reply:
@matipojo looks like It's not just an array, we still need to define this for each control. 

<img width="890" alt="CleanShot 2022-10-24 at 09 50 19@2x" src="https://user-images.githubusercontent.com/100698721/197464858-9e31ba06-5800-45c1-9d46-2099985f0203.png">

<img width="1049" alt="CleanShot 2022-10-24 at 09 51 46@2x" src="https://user-images.githubusercontent.com/100698721/197464901-06fbd486-26d4-44cd-a13b-94a6ddaa208b.png">

This logic is located in : https://github.com/elementor/elementor/blob/master/core/files/css/base.php

As I can see it's not just an array, it is more complicated than this. We still to define a style_field becouse The function call itself in eventually 

![CleanShot 2022-10-24 at 09 58 58@2x](https://user-images.githubusercontent.com/100698721/197466212-16d1b896-7827-474f-bcba-08e1a12cfff7.png)